falcon/asgi/structures.py

```diff
@@ -5,6 +5,60 @@
 
 
 class SSEvent:
```

Comment:
A finding not directly related to the proposed changeset, but there seems to be some asymmetry between parameter validation in the constructor and the `serialize()`-ation function.

Passing a parameter that does not reduce to truth in a boolean test can go unnoticed in the up front check, e.g.:
```python
>>> from falcon.asgi.structures import SSEvent
>>> event = SSEvent(text=0.0)
>>> event.serialize()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/vytas/progs/falcon/falcon/asgi/structures.py", line 85, in serialize
    block += 'data: ' + self.text + '\n'
TypeError: can only concatenate str (not "float") to str
```

Reply:
Nice catch. I'll address this in a followup PR.